src/@next/components/organisms/CartRow/types.ts

```diff
@@ -0,0 +1,52 @@
+import { IImage } from "@types";
+import React from "react";
+
+export interface IProps {
```

Comment:
This generic type name is the same as in all components in `@next` so far. You might be right but I think it might be better to fix naming convention in separate pull request.